spec/services/doc_auth/lexis_nexis/lexis_nexis_client_spec.rb

```diff
@@ -117,4 +119,85 @@
       )
     end
   end
+
+  context 'with selfie check enabled' do
+    ## enable feature
+    let(:workflow) { 'LIVENESS.CROPPING.WORKFLOW' }
+    before do
+      allow(FeatureManagement).to receive(:idv_allow_selfie_check?).and_return(true)
+    end
+    describe 'when success response returned' do
+      before do
+        stub_request(:post, image_upload_url).to_return(
+          body: LexisNexisFixtures.true_id_response_success_with_liveness,
+        )
+      end
+      it 'returns a successful response' do
+        result = client.post_images(
+          front_image: DocAuthImageFixtures.document_front_image,
+          back_image: DocAuthImageFixtures.document_back_image,
+          image_source: image_source,
+          selfie_image: DocAuthImageFixtures.selfie_image,
+          liveness_checking_required: true,
+        )
+        expect(result.success?).to eq(true)
+        expect(result.class).to eq(DocAuth::LexisNexis::Responses::TrueIdResponse)
+        expect(result.doc_auth_success?).to eq(true)
+        expect(result.selfie_live?).to eq(true)
+        expect(result.selfie_quality_good?).to eq(true)
+        expect(result.selfie_check_performed?).to eq(true)
+      end
+    end
+
+    describe 'when selfie failure response returned' do
+      before do
+        stub_request(:post, image_upload_url).to_return(
```

Comment:
This is non-blocking feedback.  I peeked at the `LexisNexisFixtures.true_id_response_failure_with_liveness` file, and most of it was created four years ago.  Are we sure that file is still up to date?

Reply:
@eileen-nava , the portrait match result part are the same.
The main differences are what they called disposition based description of some data fields, which I dont think we are using for any business purpose.

Reply:
Got it, thanks.